source/_components/vallox.markdown

```diff
@@ -0,0 +1,111 @@
+---
+layout: page
+title: "Vallox Ventilation Units"
+description: "Instructions on how to integrate Vallox ventilation units into Home Assistant."
+date: 2019-06-19 20:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category:
+  - Fan
+  - Sensor
+ha_release: 0.96
+ha_iot_class: Local Polling
+---
+
+The `vallox` component lets you control any Vallox ventilation unit that is supported by the [vallox_websocket_api](https://github.com/yozik04/vallox_websocket_api) (follow the link for a list of supported units).
+
+The component's fan platform allows you to either turn on/off the complete unit via the toggle switch or select a ventilation profile from a dropdown menu and therefore indirectly the fan speed. The four standard Vallox profiles are provided:
```

Comment:
Please update this paragraph after the change regarding fan control in the parent PR.

Reply:
Addressed